systemd_ctypes/bus.py

```diff
@@ -260,22 +277,10 @@ def default_user(attach_event=False):
         return Bus._default_user
 
     @staticmethod
-    def from_fd(fd, server_id=None, attach_event=False):
-        bus = Bus()
-        sd.bus.new(bus)
-        if server_id is not None:
-            bus.set_server(True, server_id)
-        bus.set_fd(fd, fd)
-        bus.start()
-        if attach_event:
-            bus.attach_event(None, 0)
-        return bus
-
-    @staticmethod
-    def socketpair(attach_event=False):
+    def socketpair(attach_event=True):
```

Comment:
Is it worth having `socketpair` in the API?  It is easy enough to implement explicitly in the rare cases where it is needed (such as in our tests).

Reply:
You're right.  Let's remove it.